### PR TITLE
RxJava Module

### PR DESCRIPTION
**Pull request by cammace**

### CODE REVIEW

File: services-rxbinding/src/main/java/com/mapbox/rx/api/directions/v5/EnqueueCallObservable.java [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+      if (isDisposed()) {
+        return;
+      }
+
+      try {
+        observer.onNext(response.body());
+        if (!call.isCanceled()) {
+          terminated = true;
+          observer.onComplete();
+        }
```
Comment (pakoito) [inline services-rxbinding/src/main/java/com/mapbox/rx/api/directions/v5/EnqueueCallObservable.java]: What happens if the call is cancelled? Does the observable never complete?

Comment (cammace) [reply services-rxbinding/src/main/java/com/mapbox/rx/api/directions/v5/EnqueueCallObservable.java]: hmm I might have switched up the call object coming from onResponse and the `isDisposed` methods, do you see an issue here?

Comment (pakoito) [reply services-rxbinding/src/main/java/com/mapbox/rx/api/directions/v5/EnqueueCallObservable.java]: I'm trying to understand the logic. You always get the body, even in a cancellation, but if it is cancelled it just doesn't complete the Observable, which means several operators like toList() or concatMap won't work with this API.

**Comment by cammace** [comment]
Going to go ahead and close this PR. Little discussion has occurred on our end on whether or not we still want to have a RxJava module specific to this library and until a compelling reason comes across as to why we need to support this, this PR will just continue sitting here stale. 

@zugaldia @lilykaiser @osana let me know if you guys want to sit down and discuss further actions on this.

**Pull request closed without merging**